docs/earthfile/earthfile.md

```diff
@@ -106,6 +106,16 @@ Sets a value override of `<value>` for the Dockerfile build arg identified by `<
 
 In a multi-stage Dockerfile, sets the target to be used for the build. This option is similar to the `docker build --target <target-name>` option.
 
+## LOCALLY (**experimental**)
```

Comment:
I wonder if we should include a notice like this one: https://github.com/earthly/earthly/blob/main/docs/guides/shared-cache.md#important given that this is a particularly big feature.

Reply:
good plan, I just setup a new slack channel for this.